.github/workflows/branch.yml

```
@@ -0,0 +1,16 @@
+name: nf-core branch protection
+# This workflow is triggered on PRs to master branch on the repository
+# It fails when someone tries to make a PR against the nf-core `master` branch instead of `dev`
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      # PRs are only ok if coming from an nf-core `dev` branch or a fork `patch` branch
+      - name: Check PRs
+        run: |
+          { [[ $(git remote get-url origin) == *nf-core-atacseq ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
```

That might be fine for your fork but will fail if coming from `nf-core/atacseq` which is unintended correct? 
```suggestion
          { [[ $(git remote get-url origin) == *nf-core/atacseq ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
```

---

Good spot 👍 Sorry, I assumed everything was ready and working in the template to implement GitHub Actions.

So this will need to be fixed in the template too because it came from there:
https://github.com/nf-core/tools/blob/98120e69a7d7948933d4589dd4bfa7252187320b/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/.github/workflows/branch.yml#L16

---

Or can we make that `*atacseq` instead to account for fork and the `nf-core/atacseq`? Will that work and would it be ok to do?

---

We could do that - I think thats a good idea 👍